release/package.json

```diff
@@ -34,7 +35,11 @@
   },
   "devDependencies": {
     "@babel/core": "^7.22.19",
+    "@babel/preset-env": "^7.24.8",
+    "@babel/preset-typescript": "^7.24.7",
     "@types/jest": "^29.5.5",
+    "@types/semver": "^7.5.8",
+    "@types/underscore": "^1.11.15",
```

Comment:
needed to add these explicitly so that the test can run without installing the dependencies in the parent directory